pod/perlre.pod

```diff
@@ -1348,17 +1348,17 @@ their punctuation character equivalents, however at the trade-off that you
 have to tell perl when you want to use them.
 X</p> X<p modifier>
 
-=head2 Quoting metacharacters
-
-Backslashed metacharacters in Perl are alphanumeric, such as C<\b>,
-C<\w>, C<\n>.  Unlike some other regular expression languages, there
-are no backslashed symbols that aren't alphanumeric.  So anything
-that looks like C<\\>, C<\(>, C<\)>, C<\[>, C<\]>, C<\{>, or C<\}> is
-always
-interpreted as a literal character, not a metacharacter.  This was
-once used in a common idiom to disable or quote the special meanings
-of regular expression metacharacters in a string that you want to
-use for a pattern. Simply quote all non-"word" characters:
+=head2 Quoting (or escaping) metacharacters
```

Comment:
Changing headings is perilous, but I found no instances of this being linked to directly on CPAN.

Reply:
I removed the change